From 12348acccc020dcfc04122e7821c7bfea8adde6a Mon Sep 17 00:00:00 2001
From: Faidon Liambotis <paravoid@debian.org>
Date: Tue, 21 Feb 2023 13:40:43 +0200
Subject: [PATCH 1/3] stub: Remove -g from the ESP8266 build flags

There is no debug support in the compiler. Without this, (otherwise
harmless) warnings are emitted:
  xtensa-lx106-elf-gcc: warning: target system does not support debug output
---
 flasher_stub/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/flasher_stub/Makefile b/flasher_stub/Makefile
index 55b5fa9..0bbcc5f 100644
--- a/flasher_stub/Makefile
+++ b/flasher_stub/Makefile
@@ -85,7 +85,7 @@ $(BUILD_DIR):
 
 CFLAGS = -std=c99 -Wall -Werror -Os \
          -mtext-section-literals -mlongcalls -nostdlib -fno-builtin -flto \
-         -Wl,-static -g -ffunction-sections -Wl,--gc-sections -Iinclude -Lld
+         -Wl,-static -ffunction-sections -Wl,--gc-sections -Iinclude -Lld
 CFLAGS_ESPRISCV32 = -std=c99 -Wall -Werror -Os \
 		 -march=rv32imc -msmall-data-limit=0 \
          -nostdlib -fno-builtin -flto \
-- 
2.30.2

